pkg/jx/cmd/step_create_task_test.go

```diff
@@ -48,11 +48,12 @@ func TestGenerateTektonCRDs(t *testing.T) {
 	}
 
 	cases := []struct {
-		name         string
-		language     string
-		repoName     string
-		organization string
-		branch       string
+		name           string
+		language       string
+		repoName       string
+		organization   string
+		branch         string
+		expectingError bool
```

Comment:
The expected error message includes a machine-dependent file path. We could create error types that take the path and then use a function to construct them in the test to make sure the test fails with the intended error, but this seemed good enough for now. Happy to change this up to match a specific error if desired.

Reply:
Nah, that's fine for me at least for the time being. If we ever care more specifically in the future, we can do that.